### PR TITLE
surpress compiled file warnings with silencer

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -37,3 +37,10 @@ lazy val microservice = Project(appName, file("."))
     resolvers += Resolver.jcenterRepo
   )
   .settings(PlayKeys.playDefaultPort := 9785)
+  .settings(
+    // from https://github.com/playframework/playframework/issues/7382#issuecomment-444062750
+    // silence all warnings on autogenerated files
+    scalacOptions += "-P:silencer:pathFilters=target/.*",
+    // Make sure you only exclude warnings for the project directories, i.e. make builds reproducible
```
Comment (BetaDraconis) [inline build.sbt]: What do you mean by this?

Comment (james-work-account) [reply build.sbt]: This was included in the playframework issue I linked to on L41. Would you like it removed? I'm not entirely sure what will happen without it.

Comment (BetaDraconis) [reply build.sbt]: I'm not bothered by it being there, I just didn't understand what the comment above it meant

Comment (BetaDraconis) [reply build.sbt]: Ah I see, the comment is in the source code

**Pull request merged**